proto/minder/v1/minder.proto

```diff
@@ -2072,11 +2164,23 @@ message RuleType {
         message Eval {
             // type is the type of the data evaluation.
             // Right now only `jq` is supported as a driver
-            string type = 1;
+            string type = 1 [
+                (buf.validate.field).string = {
+                    pattern: "^[a-z]+(_[a-z]+)*$",
```

Comment:
Should this use `in`, or are there more types?

Reply:
I think there are, i.e. recently we added `yq` I believe

Reply:
Refactored it to using `in` 👍